### ISSUE TITLE
Raw SQL Queries

### ISSUE THREAD

**Issue by rmad17** [reckonsys/pgorm#3]
Should the feature of executing raw sql queries be allowed? Its not a good idea but it gives someone the power the execute sql queries should an exceptional scenario arise where the normal functions are not enough. This may be an edge case but I feel would be useful. Please give your views.

**Comment by rs-ds** [comment]
Yeah - Something like [Django raw queries](https://docs.djangoproject.com/en/2.2/topics/db/sql/).  We should.
